SSA/Projects/InstCombine/tests/logs/2004-09-20-BadLoadCombine.txt

```diff
@@ -10,4 +14,10 @@
 4: "test" has unsupported operation after optimization: llvm.store
 
 4: "test" has unsupported operation after optimization: llvm.load
+<<<<<<< HEAD
+=======
+4: "test" has unsupported operation: builtin.unregistered: llvm.select
+>>>>>>> 4bf2f937 (Re-ran the sccripts)
+=======
+>>>>>>> edb64a33 (Updated tests)
```

Comment:
Do we expect these merge conflicts to be here?

Reply:
uh
I think I just did the rebase wrong
If we want to remove the extra commits, it'd be easier to just make a new branch and a new PR I think

Reply:
Lets do this then.